pages/dkp/konvoy/2.1/choose-infrastructure/awsgovcloud/prerequisites/index.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: layout.pug
+navigationTitle: Prerequisites
+title: Prerequisites
+menuWeight: 10
+excerpt: Air-gapped GovCloud installation prerequisites.
+beta: false
+enterprise: false
+---
+
+## Prerequisites
+
+Before you begin, you must have:
+
+- Linux machine (bastion) that has access to the existing VPC.
+- The `dkp` binary on the bastion.
+- [Docker][install_docker] version 18.09.2 or later installed on the bastion.
+- [kubectl][install_kubectl] for interacting with the running cluster on the bastion.
+- Valid AWS account with [credentials configured][aws_credentials].
+- VPC ID
+- Subnet IDs
+- Security Group IDs
+- Control Plane Node Instance Profile Name
+- Worker Node Instance Profile Name
+- Region
+- Container Registry Mirror
+
+### Configure AWS prerequisites
+
+1.  Follow the steps in [IAM Policy Configuration](../../iam-policies).
+
+2.  Export the AWS profile with the credentials you want to use to create the Kubernetes cluster:
+
+     ```sh
+     export AWS_PROFILE=<profile>
+     ```
+### Download the Air-gapped bundle
+The air-gapped bundle includes the following packages and tooling
+
+- Command line tooling
+- Konvoy air gapped bundle,
+- Kommander air gapped bundle,
+- Konvoy Image Builder
+- CentOS/RHEL 7.9 dependencies
+- Setup script
+- Cluster initialization script
+- Docker registry image
+
+1. Download the air-gapped bundle.
+
+```sh
+curl -O downloads.mesosphere.io/konvoy/airgapped/os-packages_1.21.6_x86_64_rpms.tar.gz
+curl -O downloads.mesosphere.io/konvoy/airgapped/pip-packages.tar.gz
```

Comment:
I think these will eventually be on the downloads.d2iq site ... don't know if we want to change this or not ... @cneth do you know?

```suggestion
curl -O downloads.mesosphere.io/konvoy/airgapped/os-packages_1.21.6_x86_64_rpms.tar.gz
curl -O downloads.mesosphere.io/konvoy/airgapped/pip-packages.tar.gz
```